astro_2.0/components/EditableContent/EditableContent.module.scss

```diff
@@ -60,6 +60,7 @@ $border-radius: 3px;
   border-bottom-left-radius: 3px;
   border-bottom-right-radius: 3px;
   display: flex;
+  gap: 12px;
```

Comment:
Unfortunately, this property will not work in Safari (
Maybe can use margin here?